eth2near/contract_wrapper/src/near_contract_wrapper.rs

```diff
@@ -81,13 +81,15 @@ impl ContractWrapper for NearContractWrapper {
             _ => Err("failed to extract current nonce")?,
         };
 
+        let num_blocks_in_batch = method_name.len() as u64;
+        let attached_gas_per_promise_in_batch = 300_000_000_000_000 / num_blocks_in_batch;
```

Comment:
I would recommend using `parse_gas!("300 T") ` from https://docs.rs/near-units/latest/near_units/

Reply:
Created issue https://github.com/aurora-is-near/rainbow-bridge/issues/770